pattern_manager/PatternParent parent
uint64[] num_points
float64[] step_sizes
float64[] lengths

---
bool success